lib/data_source/client/github.rb

```diff
@@ -29,30 +42,86 @@ def user_repos
           direction: 'desc',
           per_page: 100
         }
-        with_cache('user_repos') do
-          request('/user/repos', params)
+        responses = with_cache(:user_repos) do
+          all_user_repos = merge_multipage_results('/user/repos', params, 100)
+          all_user_repos
+        end
+
+        all_user_repos = Array.new
+        responses.each do |response|
+          all_user_repos = all_user_repos + response
         end
+        all_user_repos
       end
 
       def user_pulls
-        modifiers = org_modifiers('is:pr', 'user:@me', 'state:open', 'involves:@me')
+        if @pr_all_involve_me
+          modifiers = ['is:pr', 'state:open', "involves:#@me_account"]
+        else
+          modifiers = org_modifiers('is:pr', "user:#@me_account", 'state:open', "involves:#@me_account")
+        end
         params = search_params('', modifiers).merge(
           per_page: 100
         )
-        response = with_cache('user_pulls') do
-          request('/search/issues', params)
+        responses = with_cache(:user_pulls) do
+          all_user_pulls = merge_multipage_results('/search/issues', params, 100)
+          all_user_pulls
+        end
```

Comment:
Similar to the other comment

```suggestion
        responses = with_cache(:user_pulls) do
          merge_multipage_results('/search/issues', params, 100)
        end
```